docker/nwm_gui/app_server/Dockerfile

```diff
@@ -4,18 +4,26 @@
 ARG docker_internal_registry
 FROM ${docker_internal_registry}/dmod-py-sources as sources
 
-FROM python:3.8-slim as base
-# Update the aptitude package listing so that packages may be loaded and installed
-# Install python binaries to make pip installs a little smoother
-RUN apt-get update && apt-get install -y python3-django netcat
+FROM rockylinux:9.1
```

Comment:
Reasonable to be out of scope here, but the above + the `dnf` commands may work best as a common base image.

Reply:
Yeah, I've had similar thoughts.  We need a fairly significant overhaul to the way we arrange, tag, and manage our images, and that's already on the road map.  For now, we get some of the same benefits mostly for free from the build cache.